libpod/pod.go

```diff
@@ -304,3 +307,122 @@ func (p *Pod) GetPodStats(previousContainerStats map[string]*ContainerStats) (ma
 	}
 	return newContainerStats, nil
 }
+
+func (r *Runtime) GetPodsWithFilters(filters []string) ([]*Pod, error) {
```

Comment:
We don't want to move string filters all the way into Libpod - these are an artifact of the CLI, we don't want them in the Go API. @baude Where did we end up sticking container filters?